SavedInstances/SavedInstances.lua

```diff
@@ -2273,6 +2274,31 @@ function core:OnInitialize()
   end
   RequestRaidInfo() -- get lockout data
   RequestLFDPlayerLockInfo()
+  C_Calendar.OpenCalendar()
```

Comment:
is this necessary?

Reply:
Yes, it makes wow update event info, and not really open the calendar. Otherwise, event info may not available 5 seconds later.

Reply:
Isn't the calendar a Load on Demand addon? Can you see if `if not IsAddOnLoaded("Blizzard_Calendar") then LoadAddOn("Blizzard_Calendar") end` does the same thing?

Reply:
I tried both `LoadAddOn` and add dependencies in toc, but none of these works.

And then I turned to `C_Calendar`, but `C_Calendar.UpdateEvent()` does nothing. `CALENDAR_UPDATE_EVENT` triggers 5-6 times when first entering the world, but not reliable.

It is strange that `C_Calendar.OpenCalendar()` won't open the calendar even when the ui is fully loaded. I guess it takes the place where `UpdateEvent` should do.